src/singleinstance.py

```diff
@@ -52,7 +58,7 @@ def lock(self):
                     os.O_CREAT | os.O_EXCL | os.O_RDWR | os.O_TRUNC
                 )
             except OSError:
-                type, e, tb = sys.exc_info()
+                exp_type, e, tb_object = sys.exc_info()    # pylint: disable=unused-variable
                 if e.errno == 13:
```

Comment:
```python
except OSError as e:
    if e.errno == 13:
        ...
```

Reply:
Thank you for correcting. Will change the code as suggested.